install/terraform/open-match-build/open-match-build.tf

```diff
@@ -32,6 +32,25 @@ variable "vpc_flow_logs" {
   default     = "false"
 }
 
+variable "workload_identity_account_namespace" {
+  description = "Kubernetes namespace where the workload identity service account will take effect"
+  default     = "open-match"
+}
+
+# Defines the name of the Kubernetes service account used to auto upload the stress test result. 
+# Change kubernetes.serviceAccount for install/helm/open-match/subcharts/open-match-test/values.yaml 
+# file if you want to modify this name.
+variable "workload_identity_account_ksa" {
+  description = "Name of the Kubernetes service account which binds to the Google service account"
+  default     = "stress-test-upload-ksa"
+}
+
+# Defines the name of the Google service account used to bind with the KSA in open-match-test subchart.
+variable "workload_identity_account_gsa" {
```

Comment:
It's unclear why this is a variable. We don't plan on changing this. You can refer to a template by it's fully qualified name. There's already examples in this file. There's only going to be 1 of these service accounts and if there's more this variable cannot be used for those. It adds a layer of indirection that has no purpose.

Reply:
This comment applies to the other variables that aren't going to change either.